### ISSUE TITLE
Import without `/lib` or `/es6` paths like fp-ts now does

### ISSUE THREAD

**Issue by OliverJAsh** [devexperts/remote-data-ts#55]
As per https://github.com/gcanti/fp-ts/pull/1241, `fp-ts` now allows imports without `/lib` or `/es6` paths, e.g.

```ts
import * as O from 'fp-ts/Option';
```

instead of

```ts
import * as O from 'fp-ts/lib/Option';
```

or

```ts
import * as O from 'fp-ts/es6/Option';
```

Perhaps we could make the same change to this library. WDYT?

**Comment by raveclassic** [comment]
@OliverJAsh Unfortunately I'm not working at Devexperts anymore and I don't have write access to this repo anymore.
Ping @PalmZE

**Comment by ZUGAZ** [comment]
@OliverJAsh seem OK to me. Create PR and I will assist with merge.

**Comment by OliverJAsh** [comment]
Thanks. I will just duplicate the infrastructure used by fp-ts, e.g.: https://github.com/gcanti/fp-ts-rxjs/commit/d490d8ccee761f48d624e5e86ad19b0f8729c546.

**Comment by mlegenhausen** [comment]
@OliverJAsh any progress on this issue?

**Comment by OliverJAsh** [comment]
Sorry I haven't had a chance to look at it, I'm not sure I will anytime soon.

**Comment by ZUGAZ** [comment]
Thank you @mlegenhausen @OliverJAsh 
Merged #65. Will release soon.

**Comment by ZUGAZ** [comment]
Released in version 2.1.0
